extensions/wikia/WikiaInYourLang/modules/ext.wikiaInYourLang.js

```diff
@@ -155,6 +160,43 @@ require(
 			tracker.track(trackingParams);
 		}
 
+		function getWIYLRequestSentKey() {
+			return 'wikiaInYourLangRequestSent' + retrieveLinkTitle() + cacheVersion;
+		}
+
+		function getWIYLNotificationShownKey() {
+			return 'wikiaInYourLangNotificationShown' + retrieveLinkTitle() + cacheVersion;
+		}
+
+		function getWIYLMessageKey() {
+			return targetLanguage + 'WikiaInYourLangMessage' + retrieveLinkTitle() + cacheVersion;
+		}
+
+		function getWIYLLinkTitlesKey() {
+			return targetLanguage + 'WikiaInYourLangLinkTitles' + cacheVersion;
+		}
+
+		function saveLinkTitle(linkAddress) {
+			var linkAddressAry = linkAddress.split('/'),
+				listOfCachedTitles = '',
+				linkTitle = linkAddressAry[linkAddressAry.length - 1].length === 0 ? 'main' : linkAddressAry[linkAddressAry.length - 1];
```

Comment:
Yeah, because above you are getting the URL with `->getFullURL()` which does not URL encode slashes in MediaWiki. i.e. for http://starwars.wikia.com/wiki/Slicer/Canon:
```
> $title = GlobalTitle::newFromText( 'Slicer/Canon', NS_MAIN, 147 );

> echo $title->getFullURL();
http://starwars.wikia.com/wiki/Slicer/Canon
```

In that case, `linkAddressAry[linkAddressAry.length - 1]` would be "Canon" which would be incorrect. What you want to strip is native domain + /wiki/ from the start. Maybe since `nativeDomain` is being set as the naive value [here](https://github.com/Wikia/app/blob/36652bbe14d67/extensions/wikia/WikiaInYourLang/WikiaInYourLangController.class.php#L52), it should instead be set as `$oNativeWiki->city_url` which would then be correct URL, and this could simplified just to strip `nativeDomain + 'wiki/'`?